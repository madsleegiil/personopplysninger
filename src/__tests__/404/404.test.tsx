import "@testing-library/jest-dom";
import renderer from "react-test-renderer";
import PageNotFound from "../../pages/404/404";

describe("PageNotFound", () => {
  it("renders correctly", () => {
    const tree = renderer.create(<PageNotFound />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
