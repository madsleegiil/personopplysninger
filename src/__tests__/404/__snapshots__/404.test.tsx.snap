// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PageNotFound renders correctly 1`] = `
<div
  className="notfound__container"
>
  <h1
    className="typo-sidetittel"
  >
    404
  </h1>
</div>
`;
